### PR TITLE
#1045: [Typo] Badge (closes #1045)

### LINKED ISSUE #1045: [Typo] Badge
← #1034

## requirements
{describe the new feature}

## specs
{optional: describe technical specs to implement this feature, if not obvious}

## misc
{optional: other useful info}

### PR DESCRIPTION
**Pull request by gabro**
Closes #1045

### tests performed
unit tests

### CODE REVIEW

File: src/Badge/Badge.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  label: React.ReactNode,
+  /** tells if the badge is active (for styling purposes) */
+  active?: boolean,
+  /** an optional class name to pass to top level element of the component */
+  className?: string,
+  /** an optional style object to pass to top level element of the component */
+  style?: React.CSSProperties
+};
+
+export const Props = {
```
Comment (gabro) [inline src/Badge/Badge.tsx]: I maintained the export, but do we need it?

/cc @FrancescoCioria

Comment (giogonzo) [inline src/Badge/Badge.tsx]: probably leaving it there for some time will help with non-yet-TS projects in need of bugfixes or new features on brc

Comment (FrancescoCioria) [inline src/Badge/Badge.tsx]: yes, many projects are still using this.
the idea is: the code will be refactored in TS but projects not in TS should be able to upgrade anyway

File: package.json [hunk 2]
```diff
@@ -99,6 +99,7 @@
     "typelevel-ts": "^0.1.2"
   },
   "devDependencies": {
+    "@types/jest": "^20.0.7",
```
Comment (FrancescoCioria) [inline package.json]: watch out: `@types/x` usually don't follow semver so the `^` is very risky...

File: src/Badge/Badge.tsx (hunk 1 above)
Comment (FrancescoCioria) [inline src/Badge/Badge.tsx]: @gabro what's the inferred type of `Props`?

Comment (gabro) [inline src/Badge/Badge.tsx]: see #1049

**Pull request merged**